### PR TITLE
Incorrect schema and types used in interoperability

### LINKED ISSUE #8318: Incorrect schema for params validation in interoperability endpoints
### Expected behavior

All the params should come in JSON format and the corresponding schemas should validate against it

### Actual behavior

Interoperability endpoints are validated against incorrect schema where chainID is expected as bytes of certain length

### Steps to reproduce

Run `pos-mainchain` and call endpoint `interoperability_getChainAccount`

### Which version(s) does this affect? (Environment, OS, etc...)

`v6.0.0-alpha.17`

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #8318 #8319

### How was it solved?

- Added correct schema for endpoint request params- [🐛](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) [Incorrect schema for endpoint -](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) [Closes](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) https://github.com/LiskHQ/lisk-sdk/issues/8318
- Added correct type and removed access to invalid field - [🐛](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) [Invalid access to generatorKey property -](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) [Closes](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) https://github.com/LiskHQ/lisk-sdk/issues/8319
- Bumped versions on example apps

### How was it tested?

- Follow the [guide](https://github.com/LiskHQ/lisk-sdk/blob/development/examples/interop/README.md) to setup sidechain and mainchain, get your chain registered and observe CCU transactions happening
- Call endpoint `interoperability_getChainAccount` to check registered chains on each app
- Call endpoint `chainConnector_getSentCCUs` to check if CCUs are being sent successfully from chain connector

### CODE REVIEW

File: framework/src/modules/interoperability/schemas.ts [hunk 1]
```diff
@@ -596,25 +596,13 @@ export const getChainAccountRequestSchema = {
 	required: ['chainID'],
 	properties: {
 		chainID: {
-			dataType: 'bytes',
-			fieldNumber: 1,
```
Comment (Phanco) [inline framework/src/modules/interoperability/schemas.ts:600]: Just want to clarify if `fieldNumber` is not needed if there's only one field

**Comment by Phanco** [comment]
Apart from the question above, approved.

**Review by Phanco — APPROVED** [review]

File: framework/src/modules/interoperability/schemas.ts (hunk 1 above)
Comment (ishantiw) [reply framework/src/modules/interoperability/schemas.ts:600]: I think fieldNumber is required in codec but need to double check 🤔

File: framework/test/unit/modules/interoperability/endpoint.spec.ts [hunk 2]
```diff
@@ -102,7 +104,7 @@ describe('Test interoperability endpoint', () => {
 		lastCertificate: {
 			height: 200,
 			stateRoot: utils.getRandomBytes(32),
-			timestamp: Date.now(),
+			timestamp: Math.floor(Date.now() / 1000),
```
[the remark this answers is not in the document]
Comment (ishantiw) [reply framework/test/unit/modules/interoperability/endpoint.spec.ts:107]: I would prefer to keep minimal changes in a bug fix and only fix the part related to issue, so lets do it in some other PR that touches refactoring of tests

**Pull request merged**